kazoo/handlers/utils.py

```diff
@@ -237,11 +239,14 @@ def create_tcp_connection(
 
             # Load default CA certs
             context.load_default_certs(ssl.Purpose.SERVER_AUTH)
+            if check_hostname and not verify_certs:
+                raise ValueError(
+                    "Error, if check_hostname is specified "
+                    + "verify_certs must be False"
```

Comment:
`"Error"` is redundant, and we could probably word this a little more concisely:

```suggestion
                    "verify_certs must be False when check_hostname is specified"
```

Feel free to break across lines if the linter complains...